docs/plugins.md

```diff
@@ -130,6 +158,18 @@ Bundle the web application for production.
 
 👉 **[Back to the main docs.](/)**
 
+## TypeScript types
+
+Building your Snowpack plugin with TypeScript? You can typecheck your plugin by importing the following type:
+
+```ts
+import { SnowpackPlugin } from 'snowpack/dist-types/config';
```

Comment:
+1 for moving this into a separate "plugin-types" package, maybe as the last step of this plugin work. But this is good for now.